app/src/main/java/nl/eduvpn/app/viewmodel/BaseConnectionViewModel.kt

```diff
@@ -109,6 +133,61 @@ abstract class BaseConnectionViewModel(
         }
     }
 
+    /**
+     * @return if we should use OpenVPN
+     */
+    private suspend fun wireguard(instance: Instance, discoveredAPI: DiscoveredAPI, authState: AuthState): Boolean {
```

Comment:
I think we should give this method a better name.
Also include a sentence in the header comment about what it does.

Now it states it returns if we should use OpenVPN. That suggests that true is openvpn, false is wireguard.
But the method name suggests the opposite.
Looking at the return values does not help either.
So I think this needs some cleanup.